pages/index.tsx

```diff
@@ -30,6 +43,9 @@ const Home: NextPage = () => {
     );
   }
 
+  const prefersDarkMode = useMediaQuery('(prefers-color-scheme: dark)');
+  const darkModeElevation = prefersDarkMode ? 3 : 1;
```

Comment:
AFAIK Trends doesn't support dark mode yet. What is the purpose of this?

Reply:
I have another pending PR for dark mode. The change in elevation doesn't add much visual difference in light mode but without it in dark mode it's hard to tell where the popup ends so I thought I'd add this in to comply with the dark mode feature once implemented.